src/app/components/constitution-page/export/export.component.ts

```diff
@@ -116,7 +116,7 @@ export class ExportComponent {
   }
 
   download() {
-    const songs = Array.from(this.songs.values()).sort(compareSongASC);
+    const songs = Array.from(this.songs.values()).sort(compareObjectsFactory("title", false))
```

Comment:
Pourquoi sort par "title" ? Pour avoir l'ordre chronologie, il faut que ce soit sort par id.
```suggestion
		const songs = Array.from(this.songs.values()).sort(compareObjectsFactory("id", false));
```